PowerPointLabs/PowerPointLabs/ActionFramework/HighlightLab/HighlightText/HighlightTextActionHandler.cs

```diff
@@ -15,18 +15,19 @@ protected override void ExecuteAction(string ribbonId)
 
             if (this.GetAddIn().Application.ActiveWindow.Selection.Type == Microsoft.Office.Interop.PowerPoint.PpSelectionType.ppSelectionShapes)
             {
-                HighlightTextFragments.userSelection = HighlightTextFragments.HighlightTextSelection.kShapeSelected;
+                HighlightTextFragments.userSelection = HighlightTextFragments.HighlightTextSelection.kShapeSelected;                
             }
             else if (this.GetAddIn().Application.ActiveWindow.Selection.Type == Microsoft.Office.Interop.PowerPoint.PpSelectionType.ppSelectionText)
             {
-                HighlightTextFragments.userSelection = HighlightTextFragments.HighlightTextSelection.kTextSelected;
+                HighlightTextFragments.userSelection = HighlightTextFragments.HighlightTextSelection.kTextSelected;                
```

Comment:
Not sure why there's whitespace added.
Might want to check your ide as it seemed to have happened in another of your PRs